pages/mesosphere/dcos/services/beta-jenkins/4.0.0-2.204.2-beta/release-notes/index.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: layout.pug
+navigationTitle: Jenkins for DC/OS Beta Release Notes
+title: Jenkins for DC/OS Beta Release Notes
+menuWeight: 0
+beta: true
+excerpt: Discover the new features, updates, and known limitations in this release of the Jenkins for DC/OS Service
+enterprise: false
+--- 
+
+Jenkins 4.0.0-2.204.2
+
+## Improvements
+- Updates to Jenkins version 2.204.2 (LTS)
+- Updates [jenkins-mesos-plugin](https://github.com/jenkinsci/mesos-plugin) to 2.0.0-beta which is a major overhaul of the plugin.
+    - The jenkins-mesos-plugin has been replatformed on the [Unified Scheduler Iterface (USI)](https://github.com/mesosphere/usi) Library which provides better long-term support and more rapid delivery of Mesos features. The following features are in use:
+        - Mesos v1 API
+        - Offer-suppression
+        - [Universal Container Runtime](https://docs.d2iq.com/mesosphere/dcos/latest/deploying-services/containerizers/ucr)
+        - [Multi-Tenancy Features](/mesosphere/dcos/latest/multi-tenancy)
+- Support for Windows Agents on DC/OS 2.1
+- Support for configurable Jenkins plugins
+    - Removes previous hardcoded list of bundled plugins.
+    - A list of user desired plugins can be specified at installation time which will be installed into the service before it is started.
```

Comment:
Does this mean it downloads the plugins specified from the 'internet'?  How does this work for Airgapped installs?     If there are instructions for Airgapped customers, where do they live?    I think the docs should make it clearer where these 'come from' now that they are no longer 'bundled'.

Reply:
Added instructions for airgapped clusters.